### PR TITLE
TLS-3  Sia schema compiler for intermediate language with CLI

### PR DESCRIPTION
**Pull request by andreyluiz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,29 @@
 {
   "name": "sia-schema",
+  "version": "1.0.0",
   "packageManager": "yarn@4.5.2",
   "dependencies": {
-    "chevrotain": "^11.0.3"
+    "chevrotain": "^11.0.3",
+    "commander": "^12.1.0"
   },
+  "bin": "dist/cli.js",
```
Comment (pouya-eghbali) [inline package.json:9]: change to:

```
  "bin": {
    "sia": "dist/cli.js"
  },
```

**Review by pouya-eghbali — CHANGES_REQUESTED** [review]
Where did the `.yarn/sdks` and `.yarn/typescript` directories go? I'm asking because my vscode doesn't recognize the yarn installed types without it and it wasn't added to .gitignore by yarn.

Also, see my comment on package.json.

**Comment by andreyluiz** [comment]
@pouya-eghbali This repo was using Yarn on the Plug'n Play mode. While nice, it requires some extra configuration, like the typescript and sdks directories. I added a `.yarnrc.yml` file which disabled the Plug'n Play and uses regular `node_modules`. This renders these two directories useless. When you run `yarn install` on your machine, it will use `node_modules` as well because it's configured for it.

File: package.json (hunk 1 above)
Comment (andreyluiz) [reply package.json:9]: Should the cli also be name just `sia`?

**Review by pouya-eghbali — APPROVED** [review]

**Pull request merged**